src/os/host-start.r

```diff
@@ -678,3 +628,66 @@ comment [
     return :host-repl
 ]
 
+
+; Define REPL! object for skinning - stub for elsewhere?
+;
+
+repl!: make object! [
+    repl: true
+    counter: 0
+    last-result: _ ;-- last evaluated result (sent by HOST-REPL)
+
+    ; Called on every line of input by HOST-REPL in %os/host-repl.r
+    ;
+    cycle: does [
+        if zero? ++ counter [print-greeting] ;-- only load skin on first cycle
+        counter
+    ]
+
+    ;; APPEARANCE (can be overridden)
+
+    prompt:   {>> }
+    result:   {== }
+    warning:  {!! }
+    error:    {** }                ;; not used yet
+    info:     to-string #{e29398}  ;; info sign!
+    greeting: _
+    print-prompt:   proc []  [print/only prompt]
+    print-result:   proc []  [print unspaced [result last-result]]
+    print-warning:  proc [s] [print unspaced [warning reduce s]]
+    print-error:    proc [e] [print e]
+    print-info:     proc [s] [print [info space space reduce s]]
+    print-greeting: proc []  [boot-print greeting]
+    print-gap:      proc []  [print-newline]
+
+    ;; BEHAVIOR (can be overridden)
+    
+    input-hook: func [
+        {Receives line input, parse/transform, send back to repl eval}
+        s
+    ][
+        s
+    ]
+
+    dialect-hook: func [
+        {Receives code block, parse/transform, send back to repl eval}
+        s
+    ][
+        s
+    ]
+
+    shortcuts: make object! [
+        q: [quit]
+        list-shortcuts: [print system/repl/shortcuts]
+    ]
+
+    ;; HELPERS (could be overridden!)
+
+    add-shortcut: proc [
+        {Add/Change REPL shortcut}
+        name  [any-word!]   {shortcut name}
+        block [block!]      {command(s) expanded to}
```

Comment:
I personally do not believe in manual alignments, or the adjustments required to maintain them.  (Perhaps for some presentation contexts, like if I were doing code in a PowerPoint slide)

In the Rebol codebase I dodge the problem with:

    add-shortcut: proc [
        {Add/Change REPL shortcut}
        name  [any-word!]
            {shortcut name}
        block [block!]
            {command(s) expanded to}